core/src/epicli/data/common/ansible/playbooks/roles/prometheus/tasks/main.yml

```diff
@@ -9,10 +9,10 @@
 
 - name: Set master exists flag
   set_fact:
-    does_k8s_exists: "(groups['kubernetes_master'] | default([])) | length > 0"
+    does_k8s_exist: "{{ (groups['kubernetes_master'] | default([]) | length > 0) or k8s_as_cloud_service }}"
```

Comment:
There are places where `k8s_as_cloud_service` is checked to be defined, but it's not necessary as it's located in default settings with `false` value.